app/config.go

```diff
@@ -43,6 +44,11 @@ type UserInfo struct {
 	Subdir   *string
 }
 
+// Cors contains settings related to Cross Origin Resource Sharing (CORS)
+type Cors struct {
+	Origin string
```

Comment:
I think it would be nice and complete, if you could provide configuration options for the fields CORS fields:

- Allow-Header
- Allow Methods

Reply:
I originally thought about that, but then decided not to. The reason is that I don't see any benefit for the user really, as one needs alls the methods and headers involved in WebDAV to ensure functionalty You either want to allow CORS in your setup or you don't, and selecting the hosts to enable this for should suffice. Only allowing specific headers or methods will break the WebDAV operation. WDYT?

Reply:
Ok, you gut me